openmc/cell.py

```diff
@@ -182,6 +189,70 @@ def translation(self):
     def volume(self):
         return self._volume
 
+    @property
+    def atoms(self):
+        """ Get total number of atoms of each nuclide in the cell
```

Comment:
For documentation to render properly, the description of `property`s appear under "Attributes" in the class docstring. Can you move this there? Unfortunately that means it's not really possible to document the exceptions, but that's not a big deal since we're not doing that elsewhere.

Reply:
In Commit 2debfc0d8a56e969c76af97ac377232a2d69e744, the docstring of atoms was removed. Only the entry in "Attributes" section in main docstring of `Cell` class was left.